package.json

```diff
@@ -125,6 +123,7 @@
     "jest": "^26.6.3",
     "jest-transform-stub": "^2.0.0",
     "lint-staged": "^11.1.1",
+    "module-alias": "^2.2.2",
```

Comment:
https://github.com/ilearnio/module-alias/

0 dependencies and seems solid. Hasn't been updated in a while but I guess it's not really something that needs to change very often. Supports CJS and ESM (though at this point we're just using ESM anyway :shrug:)